octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/editor/EntityModelEditor.java

```diff
@@ -288,23 +288,22 @@ private void createHeaderPanel(Composite parent) {
         Label entityIcon = new Label(headerComposite, SWT.NONE);
         entityIcon.setLayoutData(new GridData(SWT.FILL, SWT.CENTER, false, false, 1, 1));
         entityIcon.setImage(entityIconFactory.getImageIcon(Entity.getEntityType(entityModel)));
-        Link linkEntityName = new Link(headerComposite, SWT.NONE);
+        //Link linkEntityName = new Link(headerComposite, SWT.NONE);
+        TruncatingStyledText linkEntityName = new TruncatingStyledText(headerComposite, SWT.NONE,truncatedLabelTooltip);
         linkEntityName.setLayoutData(new GridData(SWT.FILL, SWT.CENTER, true, false, 1, 1));
         Font boldFont = new Font(linkEntityName.getDisplay(), new FontData(JFaceResources.DEFAULT_FONT, 12, SWT.BOLD ));
-        linkEntityName.setForeground(foregroundColor);
+        linkEntityName.setForeground(SWTResourceManager.getColor(SWT.COLOR_LIST_SELECTION));
         linkEntityName.setFont(boldFont);
-        try {
-			linkEntityName.setText("<A>" + entityService.findEntity(Entity.getEntityType(entityModel),Long.parseLong((String) entityModel.getValue(EntityFieldsConstants.FIELD_ID).getValue())).getValue(EntityFieldsConstants.FIELD_NAME).getValue() + "</A>");
-		} catch (ServiceException e) {
-			e.printStackTrace();
-		}
-        linkEntityName.addListener(SWT.Selection, new Listener() {
+        linkEntityName.setBackground(backgroundColor);
+        linkEntityName.setText(entityModel.getValue(EntityFieldsConstants.FIELD_NAME).getValue().toString());

```

Comment:
this was supposed to be a URL. Does it still work?

Reply:
yes it is still working